ixmp/reporting/__init__.py

```diff
@@ -480,12 +480,12 @@ def aggregate(self, qty, tag, dims_or_groups, weights=None, keep=True,
         return self.add(key, comp, strict=True, index=True, sums=sums)
 
     def disaggregate(self, qty, new_dim, method='shares', args=[]):
-        """Add a computation that disaggregates *var* using *method*.
+        """Add a computation that disaggregates *qty* using *method*.
```

Comment:
please @khaeru confirm that the **qty** argument of this method could actually be any quantity (params, variables, etc.) or it has to be just a variable (you wrote **var** probably due to some reason).

Reply:
You're right, it's any quantity :+1: